apps/auth/static/app.css

```diff
@@ -0,0 +1,98 @@
+body {
+  font-family: "proxima-nova", sans-serif;
+  text-align: center;
+  font-size: 300%;
+  font-weight: 100;
+}
+pre {
+  text-align: left;
+}
+input[type=checkbox],
+input[type=radio] {
+  position: absolute;
+  opacity: 0;
+}
+input[type=checkbox] + label,
+input[type=radio] + label {
+  display: inline-block;
+}
+input[type=checkbox] + label:before,
+input[type=radio] + label:before {
+  content: "";
+  display: inline-block;
+  vertical-align: -0.2em;
+  width: 1em;
+  height: 1em;
+  border: 0.15em solid #0074d9;
+  border-radius: 0.2em;
+  margin-right: 0.3em;
+  background-color: white;
+}
+input[type=radio] + label:before {
+  border-radius: 50%;
+}
+input[type=radio]:checked + label:before,
+input[type=checkbox]:checked + label:before {
+  background-color: #0074d9;
+  box-shadow: inset 0 0 0 0.15em white;
+}
+input[type=radio]:focus + label:before,
+input[type=checkbox]:focus + label:before {
+  outline: 0;
+}
+.btn {
+  font-size: 140%;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  border: 0;
+  background-color: #16214D;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#16214D" to be "#16214d" (color-hex-case)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=5459956)